src/Components/WebAssembly/WebAssembly/src/Hosting/RootComponentMapping.cs

```diff
@@ -13,11 +14,11 @@ namespace Microsoft.AspNetCore.Components.WebAssembly.Hosting
     {
         /// <summary>
         /// Creates a new instance of <see cref="RootComponentMapping"/> with the provided <paramref name="componentType"/>
-        /// and <paramref name="selector"/>.
+        /// and <paramref name="selectorOrId"/>.
         /// </summary>
         /// <param name="componentType">The component type. Must implement <see cref="IComponent"/>.</param>
-        /// <param name="selector">The DOM element selector.</param>
-        public RootComponentMapping(Type componentType, string selector)
+        /// <param name="selectorOrId">The DOM element selector or component registration id for the component.</param>
+        public RootComponentMapping(Type componentType, string selectorOrId)
```

Comment:
Can we leave these names unchanged and update the docs? This is a unnecessary breaking change.

Reply:
Removed the breaking change